### ISSUE TITLE
MSI Installer for Windows ARM64

### ISSUE THREAD

**Issue by rstolpe** [PowerShell/PowerShell#16992]
### Summary of the new feature / enhancement

An installer for the Window ARM64 installation.
Many of us running macOS etc. has windows running under 11 ARM64 as VMs. Also ARM64 is the future ;)

**Comment by adityapatwardhan** [comment]
Wix stable version does not support win-arm64 for creating MSI. We cannot take dependency of beta versions due to compliance reasons.

**Comment by rhubarb-geek-nz** [comment]
This uses WiX stable to create an MSI for ARM64. The MSI is marked as Platform="x64" but the binaries are "arm64". It installs in "c:\Program Files\PowerShell\7", adds the path and creates a start menu shortcut.

[This script uses WiX to package the contents of the official zip as an MSI.](https://github.com/rhubarb-geek-nz/PowerShell/tree/main/win)

**Comment by rollingmoai** [comment]
Duplicate of #12116

**Comment by thijsvandien** [comment]
> Wix stable version does not support win-arm64 for creating MSI. We cannot take dependency of beta versions due to compliance reasons.

Apparently WiX v4 with ARM support is now stable.

**Comment by Xazax-hun** [comment]
Glad this is unblocked, I hope there will be some progress in this area.

**Comment by rstolpe** [comment]
> Glad this is unblocked, I hope there will be some progress in this area.

Nothing yet sadly

**Comment by rhubarb-geek-nz** [comment]
It lives!

[v7.4.3 Release of PowerShell](https://github.com/PowerShell/PowerShell/releases/tag/v7.4.3)

[PowerShell-7.4.3-win-arm64.msi](https://github.com/PowerShell/PowerShell/releases/download/v7.4.3/PowerShell-7.4.3-win-arm64.msi)
